modules/javafx.graphics/src/main/java/com/sun/javafx/sg/prism/NGRegion.java

```diff
@@ -612,7 +612,7 @@ private void renderAsShape(Graphics g) {
             Rectangle rect = null;
             // RT-25013: We need to make sure that we do not use a cached image in the case of a
             // scaled region, or things won't look right (they'll looked scaled instead of vector-resized).
-            if (cacheMode != 0 && g.getTransformNoClone().isTranslateOrIdentity()) {
+            if (cacheMode != 0 && g.getTransformNoClone().isTranslateOrIdentity() && !(g instanceof PrinterGraphics) {
```

Comment:
You are missing a closing paren, which is why the CI build failed.

Reply:
fixed